reactor-core/src/main/java/reactor/core/publisher/DirectProcessor.java

```diff
@@ -29,16 +29,27 @@
 
 /**
  * Dispatches onNext, onError and onComplete signals to zero-to-many Subscribers.
+ * <br>
+ * <br>
  * <p>
+ *     <b>Note: </b> DirectProcessor does not support backpressure as it is described in
```

Comment:
Not supporting backpressure means that the `Processor` ignores it and can overflow the `Subscriber`. A more accurate description would be: 

"DirectProcessor does not coordinate backpressure between its Subscribers and the upstream and consumes its upstream in an unbounded manner, if a downstream Subscriber is not ready to receive items (hasn't requested yet or enough), it will be terminated with an IllegalStateException. If there are no Subscribers, upstream items are dropped and only the terminal events are retained."

Reply:
Thank you for that comment!